### PR TITLE
Category orderlist page 159226163

### PR DESCRIPTION
**Pull request by jaypal412**
## Why?

**Feature:**   
1.Work on the login dropdown
2. Work on the filter UI 
3. work on category page.
4. Work on order list page.
5: Adding the option text to order success,cart,my profile order page.

## This change addresses the need by:
In this all changes are done.

[delivers #159226163]

[Story](https://www.pivotaltracker.com/story/show/159226163)

### CODE REVIEW

File: src/app/home/content/content-header/content-header.component.ts [hunk 1]
```diff
@@ -72,6 +83,19 @@ export class ContentHeaderComponent implements OnInit {
...
+    this.isfilterModalShown = flag;
+  }
+  selectedInput(newVal) {
+    this.subselectedItem = newVal;
+    console.log(this.subselectedItem)
+  }
+  childselectedInput(newVal) {
+    this.childselectedItem = newVal;
+    console.log(this.childselectedItem)
+  }
```
Comment (gopalshimpi) [inline src/app/home/content/content-header/content-header.component.ts]: remove console.log

File: src/app/home/filter-mobile-menu/filter-mobile-menu.component.spec.ts [hunk 2]
```diff
@@ -1,20 +1,20 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (gopalshimpi) [inline src/app/home/filter-mobile-menu/filter-mobile-menu.component.spec.ts]: Why did you touch test files?

Comment (jaypal412) [reply src/app/home/filter-mobile-menu/filter-mobile-menu.component.spec.ts]: This component is created by cop paste. so just changed the component name.

File: src/app/home/content/content-header/content-header.component.ts (hunk 1 above)
Comment (jaypal412) [reply src/app/home/content/content-header/content-header.component.ts]: removed

**Comment by pkrawat1** [comment]
duplicate #205

**Pull request closed without merging**